camptix.js

```diff
@@ -153,4 +165,63 @@ var docCookies={getItem:function(e){return decodeURIComponent(document.cookie.re
 			} );
 		}
 	}
+
+	/**
+	 * Lazy load camptix_attendees avatars
+	 *
+	 * @uses jQuery appear plugin
+	 * @uses Jetpack's jquery.spin
+	 */
+	var lazyLoad = {
+		cache: {
+			$document:  $(document),
+			$attendees: $('#tix-attendees')
+		},
+
+		init: function() {
+			// Dependencies
+			if (
+				lazyLoad.cache.$attendees.length < 1
+				||
+				'undefined' == typeof $.fn.appear
+				||
+				'undefined' == typeof wp
+			) {
+				return;
+			}
+
+			var spinner = 'undefined' !== typeof $.fn.spin;
+
+			lazyLoad.avatarTemplate = wp.template('tix-attendee-avatar');
+
+			lazyLoad.cache.$placeholders = lazyLoad.cache.$attendees.find('.avatar-placeholder');
+
+			lazyLoad.cache.$placeholders.appear({ interval: 500 });
+
+			lazyLoad.cache.$placeholders.one('appear', function(event) {
+				var $placeholder = $(event.target);
+
+				$placeholder.each(function() {
+					if (spinner) {
```

Comment:
I think this condition could go before `$placeholder.each(function()`, so it's only called once instead of `lazyLoad.cache.$attendees.length` times